File: Program36.c
#include<stdio.h>

///////////////////////////
///Author : Kale Arvind
///Function Name : DisplayNonFactors
//Input : 6 
//Output : 4 and 5 
/////////////////////////////


void DisplayNonFactors(int iNo)
{
    int iCnt = 0 ;

    printf("Non Factors of %d are\n",iNo);

    for(iCnt = 1 ; iCnt < iNo ; iCnt++)
    {
        if((iNo % iCnt)!= 0 )
        {
            printf("%d\n",iCnt);
        }
    }
}

int main()
{
    int iValue = 0 ;
    
    printf("Enter the Number\n");
    scanf("%d",&iValue);

    DisplayNonFactors(iValue);

    return 0 ;
}

File: Program36.py
# Input : 4 
# Output : 
# 4 
# 3 
# 2 
# 1 

def displayPattern(iValue) : 
    for i in range(iValue , 0 , -1 ):
        print(i) 

def main():
    iNo = 0  

    iNo = int(input("Enter a Number : ")) 

    displayPattern(iNo)

if __name__=="__main__":
    main()